CoreUtilities/src/au/gov/asd/tac/constellation/utilities/icon/IconData.java

```diff
@@ -171,6 +172,31 @@ private static BufferedImage scaleImage(final BufferedImage image, final int siz
         }
     }
 
+    @Override
+    public int hashCode() {
+        int hash = 7;
+        hash = 79 * hash + Arrays.hashCode(this.data);
+        return hash;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (getClass() != obj.getClass()) {
+            return false;
+        }
```

Comment:
Acknowledging this was probably auto-generated but if you're able to merge these if statements together, that would be great

Reply:
Updated